components/BridgeTokenDestination.tsx

```diff
@@ -114,27 +114,31 @@ const BridgeTokenDestination: VFC<
 					</div>
 				)}
 			</div>
-			<div css={styles.formField(bridgeAction === "Deposit")}>
-				<label htmlFor="transferCENNZAddressInput">CENNZnet ADDRESS</label>
-				<AddressInput
-					id="transferCENNZAddressInput"
-					value={transferCENNZAddress}
-					onChange={onTransferCENNZAddressChange}
-					addressType="CENNZnet"
-					ref={cennzAddressInputRef}
-				/>
-			</div>
-
-			<div css={styles.formField(bridgeAction === "Withdraw")}>
-				<label htmlFor="transferMetaMaskAddressInput">ETHEREUM ADDRESS</label>
-				<AddressInput
-					id="transferMetaMaskAddressInput"
-					value={transferMetaMaskAddress}
-					onChange={onTransferMetaMaskAddressChange}
-					addressType="Ethereum"
-					ref={metaMaskAddressInputRef}
-				/>
-			</div>
+			{bridgeAction === "Deposit" && (
+				<div css={styles.formField}>
+					<label htmlFor="transferCENNZAddressInput">CENNZnet ADDRESS</label>
+					<AddressInput
+						id="transferCENNZAddressInput"
+						value={transferCENNZAddress}
+						onChange={onTransferCENNZAddressChange}
+						addressType="CENNZnet"
+						ref={cennzAddressInputRef}
+					/>
+				</div>
+			)}
+
+			{bridgeAction === "Withdraw" && (
+				<div css={styles.formField}>
+					<label htmlFor="transferMetaMaskAddressInput">ETHEREUM ADDRESS</label>
+					<AddressInput
+						id="transferMetaMaskAddressInput"
+						value={transferMetaMaskAddress}
+						onChange={onTransferMetaMaskAddressChange}
+						addressType="Ethereum"
+						ref={metaMaskAddressInputRef}
+					/>
+				</div>
+			)}
 		</div>
```

Comment:
The `AddressInput` should use the account address that the main wallet connected to, so in case of `MetaMask`, it should be the Eth address